source/Halibut.Tests/Support/LatestClientAndLatestServiceBuilder.cs

```diff
@@ -518,7 +558,7 @@ public async Task<ClientAndService> Build(CancellationToken cancellationToken)
                 portForwarderReference.Value = portForwarder;
             }
 
-            return new ClientAndService(client, service, serviceUri, clientTrustsThumbprint, portForwarder, disposableCollection, httpProxy, httpProxyDetails, forceClientProxyType, cancellationTokenSource);
+            return new ClientAndService(pollingClient, client, service, serviceUri, clientTrustsThumbprint, portForwarder, disposableCollection, httpProxy, httpProxyDetails, forceClientProxyType, cancellationTokenSource);
```

Comment:
This feels a bit confusing to start passing a `PollingClient` when we are setting up listing stuff

Reply:
After discussion, we decided to simplify it down to a nullable `clientUri`